docs/tutorials/vehicle_model/README.md

```diff
@@ -0,0 +1,242 @@
+  - [1.4 DTDL properties](#14-dtdl-properties)
+  - [1.5 DTDL commands](#15-dtdl-commands)
+- [2. DTDL validation](#2-dtdl-validation)
+- [3. Translating DTDL to code](#3-translating-dtdl-to-code)
+
+## Introduction
+
+In this tutorial, you will learn how to create a vehicle model using [Digital Twins Definition Language (DTDL) version 3.0](https://azure.github.io/opendigitaltwins-dtdl/DTDL/v3/DTDL.v3.html), and build a corresponding Rust vehicle model. The focus will be on translating DTDL to a programming language, specifically Rust. While Rust is used here to be consistent with Ibeji's samples being in Rust, you can use any programming language.
+
+This tutorial provides a basic understanding of DTDL in the context of our Ibeji samples, but it is not a comprehensive guide to DTDL. Specifically, it covers [interfaces](https://azure.github.io/opendigitaltwins-dtdl/DTDL/v3/DTDL.v3.html#interface), [commands](https://azure.github.io/opendigitaltwins-dtdl/DTDL/v3/DTDL.v3.html#command), [properties](https://azure.github.io/opendigitaltwins-dtdl/DTDL/v3/DTDL.v3.html#property)
+
+Please refer to the [DTDL v3](https://azure.github.io/opendigitaltwins-dtdl/DTDL/v3/DTDL.v3.html) documentation for more information on DTDL.
+
+>Note: Currently in Ibeji, DTDL is used as a reference representation of the vehicle hardware. It serves as a guide to manually construct the vehicle model in code. In the future, the vehicle model code should be generated from a DTDL file.
+
+## 1. Create a reference vehicle model with DTDL
+
+In this section, you will be learning how to create a reference vehicle model with DTDL. DTDL is used to define a digital twin model of a vehicle, and its capabilities.
+
+Under the {repo-root-dir}/digital-twin-model/dtdl directory, contains sample DTDL files to describe our sample in-vehicle digital twin model. Our sample in-vehicle digital twin model consists of an airbag seat massager, camera, human machine interface (HMI), HVAC, and an on-board diagnostics (OBD).
+
+To simplify, this tutorial will guide you in creating a digital twin vehicle model with HVAC and HMI systems.
+
+### 1.1 DTDL interfaces
+
+>[DTDL v3 Interface:](https://azure.github.io/opendigitaltwins-dtdl/DTDL/v3/DTDL.v3.html#interface) "An Interface describes the contents (Commands, Components, Properties, Relationships, and Telemetries) of any digital twin".
+
+Each sample DTDL file in the {repo-root-dir}/digital-twin-model/dtdl directory, and begins with an interface at the top level.
+
+>Tip: A suggested strategy for creating your in-vehicle digital twin model is to first determine the components and vehicle characteristics you want to include in your common vehicle, followed by deciding the level of granularity you need for your in-vehicle digital twin model.
+>
+>For example, in our samples, we categorize HVAC and OBD separately. The `hvac.json` DTDL file contains all HVAC-related elements, while the `obd.json` DTDL file encompasses all OBD-related components.

```

Comment:
It might be cleaner to choose the same verb for both clauses here: either "contains" or "encompasses"